engine/runtime-parser/src/main/scala/org/enso/compiler/core/ir/module/scope/definition/Method.scala

```diff
@@ -41,14 +42,14 @@ object Method {
   /** The definition of a method for a given constructor.
     *
     * @param methodReference a reference to the method being defined
-    * @param body            the body of the method
+    * @param bodyReference   the body of the method
     * @param location        the source location that the node corresponds to
     * @param passData        the pass metadata associated with this node
     * @param diagnostics     compiler diagnostics for this node
     */
   sealed case class Explicit(
     override val methodReference: Name.MethodReference,
-    val bodySeq: Seq[Expression],
+    val bodyReference: Persistance.InlineReference[Expression],
```

Comment:
I think it's much cleaner to use a specialized type as a holder of these, instead of a general-use collection like `Seq`.